.github/workflows/test.yml

```diff
@@ -69,15 +66,6 @@ jobs:
         with:
           version: "0.3.2"
           checksum: ${{ matrix.checksum }}
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-  test-without-github-token:
```

Comment:
Now that we have a default token let's test here that supplying a token works without throwing an error

Reply:
Done in https://github.com/astral-sh/setup-uv/pull/61/commits/c02ae58b831cc26ee42f0a7215cc226777e68d99 👍🏼